frontend/src/app/patients/EditPatient.test.tsx

```diff
@@ -684,4 +684,55 @@ describe("EditPatient", () => {
       expect(screen.queryByText("Save and start test")).not.toBeInTheDocument();
     });
   });
+  describe("Start test and Save and start test from edit patient", () => {
+    const renderWithUser = (
+      elementToRender:
+        | React.ReactElement<any, string | React.JSXElementConstructor<any>>
+        | undefined
+    ) => ({
+      user: userEvent.setup(),
+      ...render(
+        elementToRender as React.ReactElement<
+          any,
+          string | React.JSXElementConstructor<any>
+        >
+      ),
+    });
+
+    it("Prompts user to start test when no edits have been made ", async () => {
+      const elementToRender = createGQLWrappedMemoryRouterWithDataApis(
+        <EditPatient
+          facilityId={mockFacilityID}
+          patientId={mockPatientID}
+          fromQueue={true}
```

Comment:
If it's from the queue we actually shouldn't expect a "Start test" button -- this relates to the bug I saw before.